cli/__init__.py

```diff
@@ -1,6 +1,14 @@
 import os
+import shutil
+import subprocess
 import sys
 
+from packaging.version import Version, parse
+
+KNOWN_PLUGINS_URL = (
```

Comment:
if this is linked to master, why not keep it here statically?

Reply:
So that no release is necessary to update this, only a merge. But as discussed in the daily, I will make it static to keep it simple.